lib/jinja2/filters.py

```diff
@@ -8,18 +8,22 @@
     :copyright: (c) 2017 by the Jinja Team.
     :license: BSD, see LICENSE for more details.
 """
-import re
 import math
-import random
+import re
 import warnings
```

Comment:
Ditto comments on changing 3rd party software (even if bundled) to reduce warnings from static analysis.  All this would make like really difficult at next Jinja2 upgrade.  However, it doesn't really matter because we'll be unbundling Jinja2 for cylc-8.  (But that doesn't really fix the security problem does it ... just punts it to external dependencies that we have no control over).  Just saying :grimacing:

Reply:
;-)